dockertest/subtestbase.py

```diff
@@ -168,6 +172,23 @@ def failif_not_in(needle, haystack, description=None):
                              (expect_s, description, haystack))
 
     @classmethod
+    def failif_not_redhat(cls, xcept=DockerTestNAError):
+        """
+        Raises an exception if this is a non-RHEL/RHELAH System
+
+        :param xcept: Non-default exception to raise
+        :raise DockerTestNAError: By default
+        """
+        try:
+            with open(cls.redhat_release_filepath, 'rb') as rhrel:
+                if 'Enterprise' not in rhrel.read():
+                    raise xcept("Expecting a Red Hat Enterprise Linux System")
```

Comment:
Everything runs on everything by default.  So the TestNA's are important.

Controlling this is really up to the caller and configuration.  So for CI, I have some changes staged (in ADEPT) that modify the ``config.ini`` exclude list based on the type of host (i.e. atomic or regular) and product (rhel or non).